src/vuln_analysis/tools/test_transitive_code_search.py

```diff
@@ -97,6 +97,29 @@ async def test_transitive_search_golang_6():
     assert len(list_path) is 0
 
 
+@pytest.mark.asyncio
+async def test_transitive_search_java_1():
+    transitive_code_search_runner_coroutine = await get_transitive_code_runner_function()
+    set_input_for_next_run(git_repository="https://github.com/cryostatio/cryostat",
+                           git_ref="8f753753379e9381429b476aacbf6890ef101438",
+                           included_extensions=["**/*.java"],
+                           excluded_extensions=["target/**/*",
+                                                "build/**/*",
+                                                "*.class",
+                                                ".gradle/**/*",
+                                                ".mvn/**/*",
+                                                ".gitignore",
+                                                "test/**/*",
+                                                "tests/**/*",
+                                                "src/test/**/*",
+                                                "pom.xml",
+                                                "build.gradle"])
+    result = await transitive_code_search_runner_coroutine("commons-beanutils:commons-beanutils:1.9.4,org.apache.commons.beanutils.PropertyUtilsBean.getProperty")
+    (path_found, list_path) = result
+    print(result)
+    assert path_found is False
+    assert len(list_path) is 1
+
```

Comment:
@tmihalac Great!
If you are having more, try to add at least more 1-2 inputs to protect your logic from future changes.

Reply:
The problem is I don't have more examples, Ronit is working on trying to provide more